c2corg_ui/static/js/auth/auth.js

```diff
@@ -55,6 +55,12 @@ app.AuthController = function($scope, appApi, appAuthentication,
    */
   this.scope_ = $scope;
 
+  if (!this.scope_['login']) {
+    this.scope_['login'] = {};
+  }
```

Comment:
I tested my solution and it works as-is; there is no need to initialize the other fields in the controller.